packages/edit-post/src/components/resize-canvas/index.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * WordPress dependencies
+ */
+import { useSimulatedMediaQuery } from '@wordpress/block-editor';
+import { useSelect } from '@wordpress/data';
+import { useEffect, useState } from '@wordpress/element';
+
+/**
+ * Function to resize the editor window.
```

Comment:
It seems like there's an undocumented return value for this React hook?

Reply:
Whoops!